app/controllers/admin/student_applications_controller.rb

```diff
@@ -32,7 +32,16 @@ def pend
   private
 
   def student_applications
-    @student_applications = StudentApplication.where(semester: @semester)
+    student_applications = StudentApplication.current
+    @application_count = student_applications.count
```

Comment:
not trying to creep but yeah, @DrakeW is right. you can even use `.size` which figures out which is more efficient for you! http://web.archive.org/web/20100210204319/http://blog.hasmanythrough.com/2008/2/27/count-length-size